rasa/core/channels/botframework.py

```diff
@@ -189,6 +189,25 @@ def __init__(self, app_id: Text, app_password: Text) -> None:
         self.app_id = app_id
         self.app_password = app_password
 
+    @staticmethod
+    def add_attachments_to_metadata(
+        postdata: Dict[Text, Any], 
+        metadata: Optional[Dict[Text, Any]]
+    ) -> Optional[Dict[Text, Any]]:
+        """ Parse the message for the attachments values to add it to metadata
```

Comment:
This sounds a little confusing - can we find a better way of phrasing this? How about something like

 ```suggestion
        """Merge the values of `postdata['attachments']` with `metadata`."""
```